### DISCUSSION TITLE
Double check in assert block

### DISCUSSION THREAD

**Discussion by JajaComp** [LemonAppDev/konsist#556] [Report a Problem]
### Tell us what version of Konsist you use.

0.12.1

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

In this location (KoDeclarationAndProviderAssertCore.kt) function invokable twice
<img width="566" alt="Снимок экрана 2023-09-16 в 23 53 35" src="https://github.com/LemonAppDev/konsist/assets/7366014/b907df57-f953-4acb-a78b-3d239f8f7df7">

### Actual behaviour

Twice invoke body of assert check

### Expected behaviour

Single invoke assert block

**Comment by igorwojda** [comment]
Good catch.
Fixed (https://github.com/LemonAppDev/konsist/pull/570), will go with the `v0.13.0` release.

**Comment by igorwojda** [comment]
Version v0.13.0 has been released.
See https://github.com/LemonAppDev/konsist/releases/tag/v0.13.0
